tests/src/base_src_test.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from starlette.testclient import TestClient
+
+from src.main import app
+
+
+class BaseSrcTest:
+    @pytest.fixture
+    def client(self):
```

Comment:
@vasylboliuk since this client is going to be used in different test it makes more sence to put it as a session/package scope fixture in to the conftest of the global test level

Reply:
```python
# unit/conftest.py
@pytest.fixture(scope="session")
def client(self) -> TestClient :
  yield TestClient(app)
```